src/nerpa_pipeline/predictions_preprocessor.py

```diff
@@ -66,6 +66,10 @@ def gen_predictions(bgc_orfs_parts, input_file_name, output_prefix, current_part
 def create_predictions_by_antiSAMSHout(antismashouts, outdir, log):
     log.info("Start create predictions by antiSMASH")
 
+    if not antismashouts:
+        log.info("Error: no antismash results found")
```

Comment:
please fix the antiSMASH spelling (in the comments it is less important, but the log messages are visible for everyone; still, it makes sense to write correctly everywhere)

Reply:
Fixed